packages/utilities/src/isEqual.ts

```diff
@@ -1,5 +1,9 @@
-// tslint:disable-next-line no-any
-export const isEqual = (itemA: any, itemB: any): boolean => {
+/**
+ * Checks if the first and second items are the NOT same recursively. Use for checking arrays and objects.
```

Comment:
You have copy and paste issues unfortunately  :(

Reply:
Also for `isNotEqual`, you probably want to fix the doc to say `are NOT the same, recursively.` instead.

Reply:
Whoops, thanks for catching that. Fixed my itchy copy+paste trigger finger.